src/main/java/ch/wisv/payments/rest/WebhookServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package ch.wisv.payments.rest;
+
+import ch.wisv.payments.model.Order;
+import lombok.extern.slf4j.Slf4j;
+import net.minidev.json.JSONObject;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Slf4j
+@Service
+public class WebhookServiceImpl implements WebhookService {
+
+    /**
+     * Notify ... of a OrderStatus change.
+     *
+     * @param order of type Order
+     */
+    @Override
+    public void sendOrderStatusChangeNotification(Order order) {
+        // Skip if no webhook url is set.
+        if (order.getWebhookUrl() == null) {
+            return;
+        }
+
+        RestTemplate restTemplate = new RestTemplate();
```

Comment:
Please use either [Apache HttpComponents or OkHttp so that Datadog can trace these calls](https://docs.datadoghq.com/tracing/setup/java/#networking-framework-compatibility) ([example from dienst2-api-client-java](https://github.com/WISVCH/dienst2-api-client-java/commit/6b6ea6f91bb7041dcbd41eba7f2df59f33a867b3)).